aladdin/bash/container/create-namespace/create-namespace

```diff
@@ -19,6 +23,9 @@ function usage {
 
 if [[ $# -eq 0 || "$1" == "-h" || "$1" == "--help" ]]; then
     usage
+elif [[ $# -eq 1 ]]; then
+	owner="$(aws sts get-caller-identity --query 'UserId' --output text | awk -F ':' '{print$2}')"
```

Comment:
I prefer `--query` over `| jq` even though it requires an extra param `--output text`

Reply:
what is the value of `owner` if the `aws` command fails? does it make the whole script fail?

Reply:
> what is the value of `owner` if the `aws` command fails? does it make the whole script fail?

the script will fail - I think we have this flaw in several places in aladdin - I went ahead and plumbed in a default owner of `nobody` so that the script will not fail and complete gracefuilly:
```
IT2771:aladdin Abbas.Bandali$ aladdin -c KDEV create-namespace bozo
...
/root/aladdin/aladdin/bash/container/create-namespace/create-namespace: line 27: abs: command not found
namespace/bozo created
namespace/bozo labeled

KDEV1115:abbas> kubectl describe namespace bozo
Name:         bozo
Labels:       owner=nobody
Annotations:  <none>
Status:       Active
```